mozilla/blurts-server
# Check subscription status using useSession for SubscriptionBadge (MNTOR-2866)

# References: 
Jira: [MNTOR-2866](https://mozilla-hub.atlassian.net/browse/MNTOR-2866)



# Description



# Screenshot (if applicable)

Not applicable.

# How to test

1. Visit `/user/dashboard` with a user that does not have Premium.
2. Give the current user Premium using `/admin/dev`.
3. After one reload of the Dashboard the subscription state should be reflected correctly.
